src/components/uri/Visualization.tsx

```diff
@@ -32,16 +32,18 @@ import { LoadingComponent, ILoadingProps } from "../simple/LoadingComponent";
 import { ErrorComponent, IErrorProps } from "../simple/ErrorComponent";
 import { IDrillableItem, generateDimensions, RuntimeError } from "../../";
 import { setTelemetryHeaders } from "../../helpers/utils";
-import { getDefaultTreemapSort } from "../../helpers/sorts";
+import { getDefaultBarChartSort, getDefaultTreemapSort } from "../../helpers/sorts";
 import { convertErrors, generateErrorMap, IErrorMap } from "../../helpers/errorHandlers";
-import { isTreemap } from "../visualizations/utils/common";
+import { isBarChart, isTreemap } from "../visualizations/utils/common";
 import { getColorMappingPredicate, getColorPaletteFromColors } from "../visualizations/utils/color";
 import { getCachedOrLoad } from "../../helpers/sdkCache";
 import { getFeatureFlags, setConfigFromFeatureFlags } from "../../helpers/featureFlags";
 import { mergeFiltersToAfm } from "../../helpers/afmHelper";
 import { _experimentalDataSourceFactory } from "./experimentalDataSource";
 import IVisualizationObjectContent = VisualizationObject.IVisualizationObjectContent;
+import { areAllMeasuresOnSingleAxis } from "../../internal/utils/mdObjectHelper";
 import { getHighchartsAxisNameConfiguration } from "../../internal/utils/propertiesHelper";
```

Comment:
I know that this is old code, but importing something from '/internal/' in UI SDK components is not good. It breaks layering of code. Some follow up to fix this would be nice

Reply:
I heard that we will do it in SDK 8, right?
https://gooddata.slack.com/archives/G6891PA6M/p1573562066004700

Reply:
yes. But even now when we want to import something from /internal/ it is ok to use relative path but it is ok to do that only in upper layers - AD/KD. It is not ok to import from app-components (/internal folder) to the code of react-components. Because react-components are layer bellow app-components

Reply:
use this as a detection of wrong direction of dependencies next time

Reply:
I create the follow up ticket here
https://jira.intgdc.com/browse/SD-718